test/e2e/subscription_e2e_test.go

```diff
@@ -1410,14 +1410,31 @@ var _ = Describe("Subscription", func() {
 		_, err := fetchCatalogSourceOnStatus(crClient, catsrc.GetName(), testNamespace, catalogSourceRegistryPodSynced)
 		require.NoError(GinkgoT(), err)
```

Comment:
I realize I can also deflate this logic into the for loop and likely cut this test cases' run time even further.